packages/utils/typescript/lib/compile.js

```diff
@@ -3,12 +3,12 @@
 const compilers = require('./compilers');
 const getConfigPath = require('./utils/get-config-path');
 
-module.exports = async (srcDir, { watch = false, configOptions = {} } = {}) => {
+module.exports = async (srcDir = "", { watch = false, configOptions = {}, compileOptions = {} } = {}) => {
   // TODO: Use the Strapi debug logger instead or don't log at all
   console.log(`Starting the compilation for TypeScript files in ${srcDir}`);
 
   const compiler = watch ? compilers.watch : compilers.basic;
-  const configPath = getConfigPath(srcDir);
+  const configPath = getConfigPath(srcDir, configOptions);
```

Comment:
Maybe here it would be more appropriate to call this `getConfigOptions` or smth similar?
Config options sound like we're overriding the content of the config file for me. What's your opinion on this?